### ISSUE TITLE
Python version

### ISSUE THREAD

**Issue by MotoRZR** [arapat/sparrow#11]
I read the excellent Arxiv paper.  Any chance of a python version coming?

**Comment by arapat** [comment]
Yes! We have a plan to adding a Python wrapper over the Rust API. Unfortunately I don't have a timeline for that yet. It is one of the first things I want to do once the implementation is more stabilized.
